src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -77,12 +77,34 @@
     <Grid>
         <TreeView Name="treeView1"
                   ItemsSource="{Binding Children}"
-                  Background="Transparent"
+                  Background="White"
                   Opacity=".5"
                   BorderBrush="{x:Null}"
                   VirtualizingStackPanel.IsVirtualizing="True"
                   VirtualizingStackPanel.VirtualizationMode="Recycling"
                   SelectedItemChanged="TreeView1_OnSelectedItemChanged">
+            <TreeView.ItemsPanel>
+                <ItemsPanelTemplate>
+                    <VirtualizingStackPanel>
+                        <VirtualizingPanel.Style>
+                            <Style TargetType="{x:Type VirtualizingPanel}">
+                                <Style.Triggers>
+                                    <DataTrigger Binding="{Binding IsOneRowContent}"
```

Comment:
If `IsOneRowContent` is true, ItemsPanel Margin is -15,5,5,5, otherwise - 5,5,5,5